package.json

```diff
@@ -47,6 +47,7 @@
         "commander": "^2.14.1",
         "ejs": "^2.5.2",
         "express": "^4.14.0",
+        "file-finder": "^0.1.2",
```

Comment:
Are we using this?

Reply:
Nope. Gave that package a try and it wasn't necessary. Removing it.